### ISSUE TITLE
[WarnSystem] Multiple issues with the temporary mutes and bans

### ISSUE THREAD

**Issue by laggron42** [retke/Laggrons-Dumb-Cogs#105]
Lately I've been seeing a lot of issues where members were not unmuted/unbanned, both on the cog support server and on my own bot. However, the traceback is not always the same, sometimes the member is simply not unmuted/unbanned, other times the whole background task breaks and needs a reload for working back. 

Anyway, I don't have much info about this, so I'll just post what I've been reported recently.

If you also have this issue, please post any traceback in the comments.

**Comment by laggron42** [comment]
Issue when unbanning people, does not interrupt the loop task.
```
[2020-06-27 00:54] [WARNING] laggron.warnsystem: [Guild GUILD_ID] Couldn't end the timed ban of Unknown#0000 (ID: None). He will stay as it is now.
Traceback (most recent call last):
  File "/data/cogs/CogManager/cogs/warnsystem/api.py", line 1167, in _check_endwarn
    await guild.unban(member, reason=reason)
  File "/data/venv/lib/python3.8/site-packages/discord/guild.py", line 1682, in unban
    await self._state.http.unban(user.id, self.id, reason=reason)
  File "/data/venv/lib/python3.8/site-packages/discord/http.py", line 225, in request
    raise HTTPException(r, data)
discord.errors.HTTPException: 400 Bad Request (error code: 50035): Invalid Form Body
In user_id: Value "None" is not snowflake.
```

**Comment by laggron42** [comment]
Issue when iterating through the temporary warnings. This **does** interrupt the loop task as soon as the cog is reloaded.

This format is the old time format used post 1.3, may be leftovers from the conversion, or, somewhere in the code, the bot still created cases in the old format.
```
[2020-06-27 00:13] [CRITICAL] laggron.warnsystem: The loop for unmutes and unbans encountered a third error. To prevent more damages, the loop will be cancelled. Timed mutesand bans no longer works for now. Reload the cog to start the loop back. If the problem persists, report the error and update the cog.
Traceback (most recent call last):
  File "/data/cogs/CogManager/cogs/warnsystem/api.py", line 1210, in _loop_task
    await self._check_endwarn()
  File "/data/cogs/CogManager/cogs/warnsystem/api.py", line 1137, in _check_endwarn
    taken_on = self._get_datetime(action["time"])
  File "/data/cogs/CogManager/cogs/warnsystem/api.py", line 155, in _get_datetime
    return datetime.fromtimestamp(int(time))
ValueError: invalid literal for int() with base 10: 'Fri 19 June 2020 11:27:00'
```

----

This same error doesn't occur only with temporary warns, but also the modlog (`[p]warnings`):
```
[2020-07-06 00:59] [ERROR] laggron.warnsystem: Exception in command 'warnings'.
 
Traceback (most recent call last):
  File "/data/venv/lib/python3.8/site-packages/discord/ext/commands/core.py", line 83, in wrapped
    ret = await coro(*args, **kwargs)
  File "/data/cogs/CogManager/cogs/warnsystem/warnsystem.py", line 880, in warnings
    time = self.api._get_datetime(case["time"])
  File "/data/cogs/CogManager/cogs/warnsystem/api.py", line 155, in _get_datetime
    return datetime.fromtimestamp(int(time))
ValueError: invalid literal for int() with base 10: 'Wed 24 June 2020 16:59:05'
```
Note: the user that provided this traceback isn't the same, and does not have any recorded issue with the loop task interrupting

**Comment by laggron42** [comment]
This *should* have been fixed by f91b304. If you still have issues, feel free to comment and I'll reopen.
